src/presentation.rs

```diff
@@ -569,18 +606,21 @@ pub(crate) trait AsRenderOperations: Debug + 'static {
     fn diffable_content(&self) -> Option<&str>;
 }
 
-/// A type that can be rendered on demand.
-pub(crate) trait RenderOnDemand: AsRenderOperations {
-    /// Start the on demand render for this operation.
+/// An operation that can be rendered asynchronously.
+pub(crate) trait RenderAsync: AsRenderOperations {
+    /// Start the render for this operation.
+    ///
+    /// Should return true iff the invocation triggered the rendering (aka if rendering wasn't
```

Comment:
Looks like a minor typo here (iff -> if)

Reply:
[see](https://en.wikipedia.org/wiki/If_and_only_if) but this is probably the second or third time someone points this out so I should just stop using this :).